### PR TITLE
column block test cases

### PR DESCRIPTION
**Pull request by jbinda**
I have added the test cases for Columns block.After me merge the Column block we should revisit to apply changes is necessary according to current state.

### CODE REVIEW

File: test-cases/gutenberg/columns.md [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+- Expect new `Column` to be added
+
+Expected look:  
+![Appender](../resources/columns-appender.png)
+
+--------------------------------------------------------------------------------
+
+##### TC004
+
+### Adding Column with Inserter (currently disabled)
```
Comment (pinarol) [inline test-cases/gutenberg/columns.md:47]: We'll need to update this case now that it is enabled

File: test-suites/gutenberg/sanity-tests.md [hunk 2]
```diff
@@ -131,3 +131,31 @@ Cover-2
 - [ ] Cover - Replace media - [steps](https://github.com/wordpress-mobile/test-cases/blob/master/test-cases/gutenberg/cover.md#tc006)
 - [ ] Cover - Gradient background is rendered properly - [steps](https://github.com/wordpress-mobile/test-cases/blob/master/test-cases/gutenberg/cover.md#tc007)
 - [ ] Cover - Gradient overlay is rendered properly - [steps](https://github.com/wordpress-mobile/test-cases/blob/master/test-cases/gutenberg/cover.md#tc008)
+
+Columns - 1
+<!--- expected behaviour --->
+- [ ] Columns - Columns block is added with default number of Column - [steps](https://github.com/wordpress-mobile/test-cases/blob/master/test-cases/gutenberg/columns.md#tc001)
+- [ ] Columns - Adding Column with AppenderButton- [steps](https://github.com/wordpress-mobile/test-cases/blob/master/test-cases/gutenberg/columns.md#tc003)
+- [ ] Columns - Adding Column with Inserter (currently disabled) - [steps](https://github.com/wordpress-mobile/test-cases/blob/master/test-cases/gutenberg/columns.md#tc004)
```
Comment (pinarol) [inline test-suites/gutenberg/sanity-tests.md:139]: Same here 
```suggestion
- [ ] Columns - Adding Column with Inserter - [steps](https://github.com/wordpress-mobile/test-cases/blob/master/test-cases/gutenberg/columns.md#tc004)
```

File: test-cases/gutenberg/columns.md (hunk 1 above)
Comment (pinarol) [reply test-cases/gutenberg/columns.md:47]: On the second thought we can also remove this test case, because it is inserter's default behavior

Comment (jbinda) [reply test-cases/gutenberg/columns.md:47]: 2nd options seems better for me as well

File: test-suites/gutenberg/sanity-tests.md (hunk 2 above)
Comment (jbinda) [reply test-suites/gutenberg/sanity-tests.md]: I have deleted it according to above comment

**Pull request merged**